src/backup/copy_job_async/copy_job/copy_job_service.py

```diff
@@ -66,17 +67,17 @@ def __schedule(source_big_query_table, target_big_query_table, job_id):
             }
         }
         try:
-            response_job_id = BigQuery().insert_job(
+            response_job_id, response_job_location = BigQuery().insert_job(
                 target_big_query_table.get_project_id(), job_data)
-            logging.info("Response job ID: " + response_job_id)
-            return response_job_id
+            logging.info("Response job ID: %s, location: %s", response_job_id, response_job_location)
+            return response_job_id, response_job_location
         except HttpError as bq_error:
             if bq_error.resp.status == 404:
                 logging.exception('404 while creating Copy Job from %s to %s' % (source_big_query_table, target_big_query_table))
-                return None
+                return None, None
             elif bq_error.resp.status == 409:
                 logging.warning('409 while creating Copy Job from %s to %s' % (source_big_query_table, target_big_query_table))
-                return job_id
+                return job_id, None
```

Comment:
In rare case, when we have specific region and timeout occur, we might not be able to retrieve this job? Could we specify it upfront during scheduling and take it from dataset?

Reply:
done - location is taken from source table metadata (at the same point when target dataset is created) and passing via task